apps/prairielearn/src/pages/partials/questionAwardedPoints.ejs

```diff
@@ -24,13 +24,20 @@
     <% } %>
     <% for (var i = 0; i < previous_variants.length; i++) { %>
         <a
-            class="badge badge-secondary <%= collapseClass %>"
+            class="badge <%= previous_variants[i].variant_id == current_variant_id ? 'badge-light' : previous_variants[i].open ? 'badge-primary' : 'badge-secondary' %> <%= collapseClass %>"
```

Comment:
I find the `.badge-light` is too easy to overlook, especially in comparison to the `.badge-primary` of the open variant. I actually think that there isn't a need to highlight the open variant, so I suggest changing the current variant to `.badge-primary` (or maybe `.badge-info`?) and removing the special handling of the open variant.

Reply:
Done in bec5ce030b8db7d0264fddae62d47d7f5a4aa10d: 
![image](https://github.com/PrairieLearn/PrairieLearn/assets/1004907/75a106d2-adaf-4272-9a45-0e7857819043)